### ISSUE TITLE
Log estimated time remaining when indexing

### ISSUE THREAD

**Issue by smcclellan** [hirosystems/ordhook#151]
Logging for the indexing process currently only lists the latest block mined:

```
05/19/2023, 07:15:43 AM | Dequeuing block #790460 for processing (# blocks inboxed: 0) |  
05/19/2023, 07:21:21 AM | Dequeuing block #790461 for processing (# blocks inboxed: 0) |  
05/19/2023, 07:37:10 AM | Dequeuing block #790462 for processing (# blocks inboxed: 0) |  
05/19/2023, 08:08:15 AM | Dequeuing block #790463 for processing (# blocks inboxed: 0) |  
05/22/2023, 08:55:13 AM | Dequeuing block #790895 for processing (# blocks inboxed: 1) |  
05/22/2023, 08:55:35 AM | Dequeuing block #790896 for processing (# blocks inboxed: 0) |
```

We often field questions about how much longer this process will take. The ask here is to add an estimated time remaining to each of these log messages.

**Comment by chresko** [comment]
@rafaelcr How much of a lift would this be?

**Comment by csgui** [comment]
@smcclellan There were a code restructuring that removed this logging from the `ordhook` codebase. Check commit 452fdd84ed8349908e73384020accd8a4b675588.

It seems that this issue can be closed.

**Comment by smcclellan** [comment]
@csgui Is the estimated time remaining being logged now?

**Comment by csgui** [comment]
@smcclellan No. The indexing only happens in the specific case of trying to repair the inscriptions db. And there is no information about the remaining time of indexing.

**Comment by csgui** [comment]
@smcclellan Anyway, is still possible to add some information about the remaining blocks on the queue to be indexed.

Any information about timing to finish the indexing is not accurate since each block has different sizes and it also depends on the network/connection speed.

Probably we can also rely on the amount of transactions inside of each block.

**Comment by 8012145393085** [comment]
Hello world

**Comment by rafaelcr** [comment]
Closing this given that estimating the time remaining for a full index is very hard to do reliably
